src/types.d.ts

```diff
@@ -179,6 +255,13 @@ declare global {
     }
     type TitleType = 'main' | 'sub';
 
+    interface ViewPanel {
```

Comment:
Maybe we should call this `PanelView` or just `View` because it sounds like a special type of a `Panel` which is not the case.

Reply:
okay, renamed it to `View`